CHANGES.rst

```diff
@@ -463,6 +463,7 @@ Other Changes and Additions
 
 - Updated wcslib to v7.1. [#9829]
 
+- Increase minimum asdf version to 2.5.2. [#9996]
```

Comment:
good point, we should have added this entry with the 2.5 change, too, so I suggest to add that other PR number there, too so users can see the reasoning why this change was made into a bugfix release.
```suggestion
- Increase minimum asdf version to 2.5.2. [#9996, #9819]
```